extensions/kotlin/deployment/pom.xml

```diff
@@ -31,7 +31,7 @@
 
         <dependency>
             <groupId>org.jetbrains.kotlin</groupId>
-            <artifactId>kotlin-compiler-embeddable</artifactId>
+            <artifactId>kotlin-compiler</artifactId>
```

Comment:
Why that change?

Reply:
This is needed otherwise the Kotlin compiler plugins can't integrate with the compiler (you get some kind of incompatibility exception).
FWIW, we probably should have had this dependency from the start.